test/utils/mocks/base.mock.ts

```diff
@@ -1,28 +1,23 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-
-import { default as sinon } from "sinon";
+import { default as sinon } from 'sinon';
 
 /**
  * A simple base class for returning mocked values
  * from a FIFO
  */
 export class BaseStackMock {
```

Comment:
@marco-ippolito this was an attempt to cage the sinon/mocha beasts. feel free to rework to more idiomatic solutions :)